resources/Microsoft.Windows.Settings/Microsoft.Windows.Settings.psm1

```diff
@@ -0,0 +1,191 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+$ErrorActionPreference = 'Stop'
+Set-StrictMode -Version Latest
+
+if ([string]::IsNullOrEmpty($env:TestRegistryPath)) {
+    $global:ExplorerRegistryPath = 'HKCU:\Software\Microsoft\Windows\CurrentVersion\Explorer\Advanced\'
+    $global:PersonalizeRegistryPath = 'HKCU:\Software\Microsoft\Windows\CurrentVersion\Themes\Personalize\'
+} else {
+    $global:ExplorerRegistryPath = $global:PersonalizeRegistryPath = $env:TestRegistryPath
+}
+
+[DSCResource()]
+class WindowsSettings {
+    # Key required. Do not set.
+    [DscProperty(Key)]
+    [string]$SID
+
+    [DscProperty()]
+    [string] $TaskbarAlignment
+
+    [DscProperty()]
+    [string] $AppColorMode
+
+    [DscProperty()]
+    [string] $SystemColorMode
+
+    [DscProperty()]
+    [bool] $RestartExplorer = $false

```

Comment:
Nothing is setting this property in the code (unless it's handled by an external process) and the Test for this property always returns InDesiredState = True. Shouldn't this be a hidden property in that case?